### PR TITLE
Update file manager configuration page

### LINKED ISSUE #9: Update the file manager configuration page to automatically attach to all mount points
Right now we provide a drop down to users to select a mount point. I've recently discovered you can simply attach the lambda to all mount points without issue. We should do this instead of forcing the user to make a selection.

### LINKED ISSUE #22: FileSystem page rebuilt so users can easily mount new EFS filesystems.

### PR DESCRIPTION
**Pull request by brandold**
*Issue #, if available:*

Closes #9
Closes #22 

*Description of changes:*

Change the file manager configuration page to:

1. Connect to all available mount targets
2. Accept a UID / GID for setting up the access point
3. Accept a path for setting up the access point

Also small update to the "creating" dialog in the filesystems page to show a tooltip when hovered over. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/web/src/routes/Configure.vue [hunk 1]
```diff
@@ -45,44 +57,59 @@ export default {
...
-          let tmpNetinfo = {}
+          let subnets = []
+          let securityGroups = []
+        
           for (var i=0, n=netinfo.length; i < n; ++i ) {
-              let netinfoKeys = Object.keys(netinfo[i])
-              let mountTargetName = netinfoKeys[0]
-              tmpNetinfo[mountTargetName] = netinfo[i][mountTargetName]
-              
+              if (i === 5) { break; }
```
Comment (eggoynes) [inline source/web/src/routes/Configure.vue:94]: Why does the variable i have to equal 5. Just curious.

**Review by eggoynes — COMMENTED** [review]
Looks good, I just added one comment.

File: source/web/src/routes/Configure.vue (hunk 1 above)
Comment (brandold) [reply source/web/src/routes/Configure.vue:94]: I discovered that there is a maximum number (5) of mount targets that a lambda can be connected to.

Comment (eggoynes) [reply source/web/src/routes/Configure.vue:94]: 👍

**Pull request merged**